src/quantum/config.js

```diff
@@ -62,4 +62,40 @@ const CONFIG = {
 
 };
 
-export default CONFIG;
+const PAGES = {
+  header:
+    ['title', 'label', 'frameworkId', 'platform', 'option', 'project', 'suite', 'extraOptions'],
+
+  data: [
+
+    ['Tp6: Facebook', 'Firefox', 10, 'windows7-32', 'pgo', 'mozilla-central', 'raptor-tp6-facebook-firefox'],
```

Comment:
If in the future you want to make these rows shorter you can use some constants like: `MC`, `WIN7` & `FX`.

Reply:
yes, maybe. Something is lost when you move data from a table into code.  Anyway, this config file has changed to handle 32 and 64 bit